package.json

```diff
@@ -63,7 +63,7 @@
     "@typescript-eslint/eslint-plugin": "^4.33.0",
     "@typescript-eslint/parser": "^4.33.0",
     "canvas": "^2.11.0",
-    "eslint": "^7.31.0",
+    "eslint": "^7.32.0",
```

Comment:
Why is this being updated?

Reply:
not sure why this updated but it should be fine. It's a minor version update which wont include breaking changes.